docs/model_configuration.md

```diff
@@ -106,6 +106,15 @@ For models that do not support batching, or do not support batching in
 the specific ways described above, *max_batch_size* must be set to
 zero.
 
+When a model is using the auto-complete feature, a default maximum batch size may 
+may be set by using the `--backend-config=default-max-batch-size=<int>` command line switch.
```

Comment:
command line argument. Switch suggests turning something on or off

Reply:
Changed "argument" to "switch"